RecoLocalCalo/Configuration/python/RecoLocalCalo_EventContent_cff.py

```diff
@@ -79,7 +79,8 @@ def _updateOutput( era, outputPSets, commands):
 phase2_hgcal.toModify( RecoLocalCaloAOD, outputCommands = RecoLocalCaloAOD.outputCommands + ['keep *_HGCalRecHit_*_*','keep *_hgcalLayerClusters_*_*'] )
 
 from Configuration.Eras.Modifier_pA_2016_cff import pA_2016
-pA_2016.toModify( RecoLocalCaloAOD.outputCommands,
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+(pA_2016|pp_on_AA_2018).toModify( RecoLocalCaloAOD.outputCommands,
```

Comment:
All other modifications of the event content for rhe pp_on_AA_2018 era modify at the same time the AOD, the RECO and the FEVT event content, see e.g. 
https://cmssdt.cern.ch/lxr/search?%21v=CMSSW_10_3_X_2018-08-31-2300&_filestring=EventContent&_string=pp_on_AA_2018

Here you only add ZDC digis to the AOD event content: is this really what you plan for that era?

Reply:
You are right. I plan to add ZDC digis to AOD, RECO and FEVT.
Just pushed another commit.